packages/tradershub/src/constants/password.ts

```diff
@@ -0,0 +1,8 @@
+export const passwordRegex = {
+    hasLowerCase: /[a-z]/,
```

Comment:
You made the regex more understandable this way for the developers but I do not think all of these `key` `values` are needed since we only use the `isPasswordValid`. I think if you leave a comment for each just like you did here would make more sense.

Reply:
I added it as simple regex for now.
This is going to change this is the base version.